voting/src/general_meeting.rs

```diff
@@ -225,6 +226,17 @@ impl GeneralMeetingManager for Context {
         };
         Ok(vote_box)
     }
+
+    fn update_vote_box(&mut self, meeting_id: &GeneralMeetingId, vote_box: &VoteBox) -> Result<(), Error> {
```

Comment:
`get_vote_box` receives `&[u8]` as key. `update_vote_box` receives `meeting_id: &GeneralMeetingId` as key.
Please use a consistent type to find a vote box.

IMO, `meeting_id: &GeneralMeetingId` is better than `&[u8]`. How about making `get_vote_box` to receive `&GeneralMeetingId` as argument instead of `key: &[u8]`?